lib/git-manager.js

```diff
@@ -6,20 +6,32 @@ var GithubApi = require('github'),
       debug: config.debug,
       version: '3.0.0'
     }),
-    getNextPage = Promise.denodeify(github.getNextPage.bind(github)),
     Pull = require('../models/pull'),
     Comment = require('../models/comment'),
     Label = require('../models/label'),
     Status = require('../models/status'),
-    Signature = require('../models/signature');
+    Signature = require('../models/signature')
+    rateLimit = require('./rate-limit.js');
 
 github.authenticate({
    type: 'oauth',
    token: config.github.token
 });
 
-var get    = Promise.denodeify(github.pullRequests.get);
-var getAll = Promise.denodeify(github.pullRequests.getAll);
+function denodeify(func) {
```

Comment:
I wonder if we should change this function name to show that we are running these functions through rateLimit object